README.md

```diff
@@ -49,11 +49,12 @@ At present, RadonDB MySQL has supported the deployment of MySQL high availabilit
 ## License
 
 RadonDB MySQL is released under the Apache 2.0, see [LICENSE](./LICENSE).
+
 ## Discussion and Community
 
 - Forum
 
-The RadonDB MySQL topic in [Kubesphere Community](https://kubesphere.com.cn/forum/t/radondb).
+The RadonDB MySQL topic is in [Kubesphere Community](https://kubesphere.com.cn/forum/t/radondb).
```

Comment:
```suggestion
    The RadonDB MySQL topic is in [Kubesphere Community](https://kubesphere.com.cn/forum/t/radondb).
```

The description should be under the list. 

Also, the line 60&62 in README.md  and the line 57&61&63 in README_zh.md should be adjusted.